rpi5.yaml

```diff
@@ -120,12 +120,19 @@ components:
         - "../meta-xt-rpi5-prod-devel"
 
       target_images:
+        - "tmp/deploy/images/%{MACHINE}/u-boot-%{MACHINE}.bin"
+        - "tmp/deploy/images/%{MACHINE}/boot.scr"
```

Comment:
It is unclear why are you adding those files. Commit message does not mention it.